tracer/test/Datadog.Trace.TestHelpers/MockTracerAgent.cs

```diff
@@ -574,7 +607,22 @@ private void HandlePotentialRemoteConfig(MockHttpParser.MockHttpRequest request)
             }
         }
 
-        private byte[] GetResponseBytes(string body)
+        private string GetStatusString(int status)
+        {
+            switch (status)
+            {
+                case 200:
+                    return "200 OK";
+                case 404:
+                    return "404 Not Found";
+                case 500:
+                    return "500 Internal Server Error";
+                default:
+                    return status.ToString();
+            }
+        }
```

Comment:
nit: just because I love switch expressions 🤣 
```suggestion
        private string GetStatusString(int status) => status switch
        {
            200 => "200 OK",
            404 => "404 Not Found",
            500 => "500 Internal Server Error",
            _ => status.ToString(),
        }
```

Reply:
Cool!